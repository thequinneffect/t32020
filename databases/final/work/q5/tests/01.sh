psql music -c 'select * from q5() order by "group";'
