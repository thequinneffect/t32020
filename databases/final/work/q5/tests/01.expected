        group         |                  genres                  
----------------------+------------------------------------------
 After the Sea        | hip-hop,post-rock,rock
 After the Sky        | 
 All of the Floyd     | rock
 All of the Sea       | hip-hop,pop,rock
 Black Eyed Boy       | country,hip-hop,rock
 Black Eyed Nails     | metal,pop,rock
 Black Mountains      | blues,rock
 Black Vanilla        | pop,rock
 Blue Floyd           | hip-hop,metal,pop,post-rock,rock
 Blue Mountains       | country,post-rock,rock
 Bring me the Joe     | hip-hop,pop,rock
 Bring me the Sea     | country,hip-hop,post-rock,rock
 Crying Feelings      | 
 Crying Gang          | country,rock
 Grand Floyd          | country,hip-hop,pop,post-rock
 Grand Forties        | country,hip-hop,metal,rock
 Grand Horde          | country,pop
 Grand Ocean          | blues,country,hip-hop,rock
 Liquid Bricks        | pop,rock
 Massive Knights      | pop,rock
 Massive Mountains    | blues,country,pop,post-rock,rock
 Nine Inch Funk       | country,pop,rock
 Nine Inch Gang       | hip-hop,metal,rock
 Nine Inch Horde      | country,hip-hop,metal,pop,post-rock,rock
 Nine Inch Peas       | rock
 Nine Inch Sea        | country,hip-hop,rock
 Nine Inch Sky        | blues,country,hip-hop,metal
 Nine Inch Stones     | blues,metal,rock
 Pink Oysters         | metal,post-rock,rock
 Poor Funk            | country,hip-hop,metal,pop,rock
 Poor Knights         | blues
 Poor Oysters         | blues,post-rock,rock
 Raging Horde         | blues,rock
 Raging Knights       | rock
 Roaring Boy          | country,hip-hop,metal,pop,post-rock,rock
 Roaring Buffalo      | country,hip-hop,pop,rock
 Roaring Empire       | rock
 Roaring Floyd        | blues,post-rock
 Roaring Funk         | blues,hip-hop,pop,rock
 Roaring Gang         | hip-hop,rock
 Roaring Sea          | post-rock,rock
 The Horde            | metal,post-rock,rock
 Velvet Oysters       | blues,metal,pop,rock
 Velvet Sky           | blues,post-rock
 We lost the Bricks   | blues,country,hip-hop,pop,rock
 We lost the Effect   | blues,hip-hop,metal,pop,rock
 We lost the Empire   | metal,rock
 We lost the Feelings | country,pop,rock
 We lost the Nails    | blues,country,hip-hop
 We lost the Sky      | blues,post-rock,rock
(50 rows)
